Wrapping/Generators/SwigInterface/igenerator.py

```diff
@@ -236,24 +236,24 @@ def warn(self, identifier, msg, doWarn=True):
             self.warnings.add((identifier, msg))
             if self.verbose:
                 if self.options.warningError:
-                    print("error(%s): %s" % (str(identifier), msg), file=sys.stderr)
+                    print(f"error({str(identifier)}): {msg}", file=sys.stderr)
```

Comment:
Does this impose Python 3.6 requirement on developer machines (for debugging and building wheels) or for everybody (pip installing)?

Reply:
Python 3.6+ is already a requirement. The answer is yes, but it is not a new requirement.

Reply:
Well, perhaps my memory is wrong here.  I thought that 3.6 was a minimum requirement, but perhaps it's 3.5?

Reply:
@dzenanz   Discussion posted:  
https://discourse.itk.org/t/set-minimum-python-version-to-3-6/3528

Reply:
👍  for moving to 3.6 as a minimum requirement.